crit/explore.go

```diff
@@ -373,3 +373,122 @@ func (c *crit) ExploreRss() ([]*RssMap, error) {
 
 	return rssMaps, nil
 }
+
+// Sk represents the sockets associated with a single process
+type Sk struct {
+	PId     uint32    `json:"pId"`
+	Sockets []*Socket `json:"sockets"`
+}
+
+// Socket represents a single socket
+type Socket struct {
+	Fd       uint32 `json:"fd"`
+	Type     string `json:"type"`
```

Comment:
It might be better to use `FdType` here to avoid confusion between the fdInfo *type* and socket *type*.